pages/contact.html

```diff
@@ -41,5 +41,18 @@
     </a>
 </div>
 </div>
+
+<div class="ui main text container">
+    <h1 class="ui header">School Contact Details</h1>
+    <p>Email School: info@mountsioncbssecondary.ie</p>
```

Comment:
Might be worth making the email address clickable, `<p>Email School: <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>`
That should do the trick, it will also bring people to their own email account with the email address added.